DEPS

```diff
@@ -19,8 +19,8 @@ deps = {
   "components/brave_sync/extension/brave-crypto": "https://github.com/brave/crypto@7e391cec6975106fa9f686016f494cb8a782afcd",
   "vendor/bat-native-ads": "https://github.com/brave-intl/bat-native-ads.git@e8685baa659e9167edf77425fc322f3b7ef5d48c",
   "vendor/bat-native-usermodel": "https://github.com/brave-intl/bat-native-usermodel.git@c3b6111aa862c5c452c84be8a225d5f1df32b284",
-  "vendor/challenge_bypass_ristretto_ffi": "https://github.com/brave-intl/challenge-bypass-ristretto-ffi.git@d6a44420b0abeb0923bfc99eac2c22e1ba86ccab",
-  "vendor/bat-native-confirmations": "https://github.com/brave-intl/bat-native-confirmations.git@1f063972b4d39376546c273ddf0423f3671a689b",
+  "vendor/challenge_bypass_ristretto_ffi": "https://github.com/brave-intl/challenge-bypass-ristretto-ffi.git@1890958a68d305ef5c51c47eb0415c3b5f53c124",
```

Comment:
these are currently pointing to `fix-linking-error` branches and should be updated before merging